bazel_tools/scala-fail-jmh-build-on-error.patch

```diff
@@ -0,0 +1,46 @@
+diff --git a/src/scala/io/bazel/rules_scala/jmh_support/BenchmarkGenerator.scala b/src/scala/io/bazel/rules_scala/jmh_support/BenchmarkGenerator.scala
```

Comment:
That looks good to me, would be great to upstream to https://github.com/bazelbuild/rules_scala

Reply:
Will do!

Reply:
Opened: https://github.com/bazelbuild/rules_scala/pull/977